yuptoo/processor/utils.py

```diff
@@ -58,6 +59,22 @@ def download_report(consumed_message):
         )
 
 
+def tracker_message(request_obj, status, status_msg):
+
+    message = {
+        "account": request_obj.get("account"),
+        "org_id": request_obj.get("org_id"),
+        "request_id": request_obj.get("request_id"),
+        "payload_id": request_obj.get("request_id"),
+        "service": "yuptoo",
+        "status": status,
+        "status_msg": status_msg,
+        "date": datetime.datetime.now().isoformat(),
+    }
+
+    return message
```

Comment:
```suggestion
def tracker_message(request_obj, status, status_msg):
    return {
        "account": request_obj.get("account"),
        "org_id": request_obj.get("org_id"),
        "request_id": request_obj.get("request_id"),
        "payload_id": request_obj.get("request_id"),
        "service": "yuptoo",
        "status": status,
        "status_msg": status_msg,
        "date": datetime.datetime.now().isoformat(),
    }

     
```

Reply:
@patilsuraj767 - one more thing about date value do we want to consider `utcnow()` or `now()`?

Reply:
I have copied it as is from puptoo.